JenkinsfileStageTrigger

```diff
@@ -139,7 +151,8 @@ pipeline {
                                     string(name: 'RPMBUILD_TAG', value: tagMap['rpmbuild']),
                                     string(name: 'RSYNC_TAG', value: tagMap['rsync']),
                                     string(name: 'OSTREE_COMPOSE_TAG', value: tagMap['ostree-compose']),
-                                    string(name: 'PACKAGE_TEST_TAG', value: tagMap['package-test'])
+                                    string(name: 'PACKAGE_TEST_TAG', value: tagMap['package-test']),
```

Comment:
You are going to have to rebase at some point. package-test is deprecated, it is all singlehost-test, so we wouldn't want that line being added back in